test-framework/remote/src/main/java/org/keycloak/testframework/RunTestOnServer.java

```diff
@@ -0,0 +1,31 @@
+package org.keycloak.testframework;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+
+import org.keycloak.common.VerificationException;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.testframework.remote.providers.runonserver.RunOnServer;
+
+public class RunTestOnServer implements RunOnServer {
```

Comment:
this file should be placed into: test-framework/remote-providers/src/main/java/org/keycloak/testframework/remote/providers/runonserver to correlate with RunOnServer stuff